raspberrypi3-aplus.coffee

```diff
@@ -0,0 +1,36 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
```

Comment:
Is there any reason why we need a new device type of this board? Shouldn't it work with a rpi3 one?

Reply:
Wasn't sure about that. Pushed again only the necessary changes.